docs/articles/streams/basics.md

```diff
@@ -210,27 +213,25 @@ The user of the library does not have to write any explicit back-pressure handli
 and dealt with automatically by all of the provided Akka Streams processing stages. It is possible however to add
 explicit buffer stages with overflow strategies that can influence the behaviour of the stream. This is especially important
 in complex processing graphs which may even contain loops (which *must* be treated with very special
-care, as explained in [Graph cycles, liveness and deadlocks](workingwithgraphs.md#graph-cycles-liveness-and-deadlocks)).
+care, as explained in [Graph cycles, liveness and deadlocks](xref:streams-working-with-graphs#graph-cycles-liveness-and-deadlocks)).
 
 The back pressure protocol is defined in terms of the number of elements a downstream ``Subscriber`` is able to receive
 and buffer, referred to as ``demand``.
 The source of data, referred to as ``Publisher`` in Reactive Streams terminology and implemented as ``Source`` in Akka
 Streams, guarantees that it will never emit more elements than the received total demand for any given ``Subscriber``.  
 
 > [!NOTE]
-> The Reactive Streams specification defines its protocol in terms of ``Publisher`` and ``Subscriber``.
-> These types are **not** meant to be user facing API, instead they serve as the low level building blocks for different Reactive Streams implementations.
+> The Reactive Streams specification defines its protocol in terms of ``Publisher`` and ``Subscriber``. These types are **not** meant to be user facing API, instead they serve as the low level building blocks for different Reactive Streams implementations.
 > Akka Streams implements these concepts as ``Source``, ``Flow`` (referred to as ``Processor`` in Reactive Streams)
-and ``Sink`` without exposing the Reactive Streams interfaces directly.
-> If you need to integrate with other Reactive Stream libraries read [Integrating with Reactive Streams](integration.md#integrating-with-reactive-streams).
+and ``Sink`` without exposing the Reactive Streams interfaces directly. > If you need to integrate with other Reactive Stream libraries read [Integrating with Reactive Streams](integration.md#integrating-with-reactive-streams).
```

Comment:
Doesn't need this link to be updated too? Also maybe put a line break for the quote